nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-processors/src/main/java/org/apache/nifi/processors/iceberg/converter/GenericDataConverters.java

```diff
@@ -48,10 +51,28 @@
 public class GenericDataConverters {
 
     static class SameTypeConverter extends DataConverter<Object, Object> {
```

Comment:
Since the functionality is changed, the method should be renamed to reflect its current function.